src/simulations/predictions/prediction_utils.jl

```diff
@@ -139,32 +139,44 @@ end
 Solve the Shallow Ice Approximation iceflow PDE for a given temperature series batch out-of-place.
 """
 function batch_iceflow_PDE(glacier_idx::I, simulation::Prediction) where {I <: Integer}
-    
     model = simulation.model
     params = simulation.parameters
     glacier = simulation.glaciers[glacier_idx]
 
+    # for now we don't use θ
+    θ = nothing
+
```

Comment:
Is this really necessary? If the definition of `\theta` is just required in order to make the call to function like `build_callback`, I will suggest instead dealing with the non-existing \theta with multiple distpatch, defining instead
```
build_callback(model.iceflow, simulation.cache.iceflow, glacier_idx) = build_callback(model.iceflow, simulation.cache.iceflow, glacier_idx, nothing)
```
Maybe I am missing a detail here, but this is for the PDE, so seems no \theta should be invoked here. You can also just use nothing as argument, unless you expect theta here to be something in the future.